content/docs/operating/configuration.md

```diff
@@ -220,13 +220,14 @@ A `tls_config` allows configuring TLS connections.
 ```
 
 
-### `<target_group>`
+### `<static_config>`
 
-A `target_group` allows specifying a list of targets and a common label set for them.
-They are the canonical way to specify static targets in a scrape configuration.
+A `static_confic` allows specifying a list of targets and a common label set
```

Comment:
Are you referring to documenting the backwards-compatibility, or do you want a note like "This option was called `target_group` up to 0.19?
In which case the same should be done for the file SD, no?